server/datastore/mysql/migrations/tables/20220608113128_AddTransparencyURLToAppConfig.go

```diff
@@ -0,0 +1,51 @@
+package tables
+
+import (
+	"database/sql"
+	"encoding/json"
+
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20220608113128, Down_20220608113128)
+}
+
+func Up_20220608113128(tx *sql.Tx) error {
```

Comment:
Input on whether this migration approach makes sense would be very much appreciated